components/compliance-service/scripts/grpc.sh

```
@@ -24,16 +25,16 @@ function cleanup() {
 trap cleanup EXIT
 
 for i in $(find components/compliance-service -name '*.proto') ; do
+  printf 'GEN: %s\n' "${i}"
   protoc -I /src \
-    -I src/components/compliance-service/api \
     -I vendor \
     -I vendor/github.com/grpc-ecosystem/grpc-gateway \
     -I vendor/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
     --go_out=plugins=grpc,paths=source_relative:/src \
     --grpc-gateway_out=request_context=true,logtostderr=true:"${fauxpath}" \
     --a2-config_out=paths=source_relative:/src \
-    "${i}"
-  printf 'GEN: %s\n' "${i}"
+    "${i}" \
+    || exit $?
```

Fail fast if any proto compilation fails.